modules/claims_api/lib/claims_api/v2/disability_compensation_evss_mapper.rb

```diff
@@ -65,10 +65,14 @@ def transform_disability_values!(disability)
         disability.delete(:serviceRelevance) if disability&.dig(:serviceRelevance).blank?
         disability.delete(:classificationCode) if disability&.dig(:classificationCode).nil? # blank is ok
 
+        disability[:name] = disability[:name].gsub(/\[|\]/, '') if disability[:name].present?
+
         if disability&.dig(:secondaryDisabilities).present?
           disability[:secondaryDisabilities] = disability[:secondaryDisabilities]&.map do |secondary|
             secondary.delete(:classificationCode) if secondary&.dig(:classificationCode).nil? # blank is ok
 
+            secondary[:name] = secondary[:name].gsub(/\[|\]/, '') if secondary[:name].present?
```

Comment:
think this is currently confirming the need for the strip here. EVSS Docker should be updated to handle `[]` now.

Reply:
Per Slack convo, we are still getting an error from EVSS when forwarding brackets. Plan is to strip brackets for now and remove these lines after confirming that EVSS accepts brackets.